core/templates/pages/topic-editor-page/modal-templates/create-new-subtopic-modal.component.html

```diff
@@ -46,8 +46,9 @@ <h3>
         </div>
         <div class="description-editor protractor-test-subtopic-description-editor" *ngIf="schemaEditorIsShown">
           <div class="oppia-editor-card-body" >
-            <schema-based-editor [schema]="getSchema.bind(this)"
+            <schema-based-editor [schema]="getSchema()"
                                  class="protractor-test-create-subtopic-page-content"
+                                 [localValue]="htmlData"
```

Comment:
Why is it localValue here and (local-value) in the next file?

Reply:
This file is an angular template and other one is an angularjs template. So, syntax differs.